copier.yml

```diff
@@ -55,69 +68,90 @@ open_source_license:
 copyright_year:
   type: str
   help: "Copyright year(s)"
-  default: "2022"
+  default: "{% now 'utc', '%Y' %}"
 
 documentation:
   type: bool
   help: "Generate documentation?"
   default: yes
 
-has_cli:
+setup_venv:
   type: bool
-  help: "Does the project have a CLI?"
-  default: no
+  help: "Setup a virtual environment?"
+  default: yes
```

Comment:
Why bother with venv creation? Thsi template uses Poetry, which manages venv automatically. Would be similar if/when it's moved to `uv`...

Reply:
I'm starting to think that for advanced users like you, this should rather be considered a temporal virtual environment just to support all operations up to pushing to GitHub, in case pre-commit and poetry are not installed globally, rather than used to develop projects. Of course you can switch Python versions using `poetry env` etc. It is quite difficult to write cross-platform commands to access the right python using `poetry shell`.